### PR TITLE
Add: Create nl-BE (Flemish)

### PR DESCRIPTION
**Pull request by carsonschildt**
These are the added translations for the Flemish language.

### CODE REVIEW

File: src/public/assets/lang/nl-BE [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Legal: Juridisch
+License: Licentie
+EULA:
+History of Incidents: Geschiedenis van incidenten
+Privacy and Security: Privacy en veiligheid
+Inspired by <%0|Avdan|%0>: Geïnspireerd doo <%0|Avdan|%0>
+AvdanOS contributors: AvdanOS medewerkers
+
+# alert.tsx
+<%0|AvdanOS|%0> is still in development.: AvdanOS is nog steeds in ontwikkeling
```
Comment (Froxcey) [inline src/public/assets/lang/nl-BE:24]: A styling mark is missing here

**Review by Froxcey — CHANGES_REQUESTED** [review]
Hey, thank you very much for the translation. Just one tiny change and we should be good to go.

File: src/public/assets/lang/nl-BE (hunk 1 above)
Comment (cruxschildt) [reply src/public/assets/lang/nl-BE:24]: Hello Froxcey,

Thanks for pointing that flaw out... I will make sure to resolve it.

**Review by Sammy99jsp — APPROVED** [review]
LGTM! I think Froxcey's busy right now, so I'm gonna merge this myself.

**Comment by Sammy99jsp** [comment]
Thank you for your translations!

**Pull request merged**